ui/app/templates/components/console/command-input.hbs

```diff
@@ -9,27 +9,21 @@
   {{else}}
     <Chevron />
   {{/if}}
-  <input onkeyup={{action "handleKeyUp"}} value={{this.value}} autocomplete="off" spellcheck="false" />
-  <ToolTip @horizontalPosition="auto-right" @verticalPosition="above" as |d|>
-    <d.Trigger>
-      <Hds::Button
-        class="hds-side-nav__icon-button"
-        {{on "click" (action "fullscreen")}}
-        data-test-tool-tip-trigger
-        @icon={{if this.isFullscreen "minimize" "maximize"}}
-        @text={{if this.isFullscreen "Minimize" "Maximize"}}
-        @isIconOnly={{true}}
-      />
-    </d.Trigger>
-    <d.Content @defaultClass="tool-tip">
-      <div class="box">
-        {{#if this.isFullscreen}}
-          Minimize
-        {{else}}
-          Maximize
-        {{/if}}
-      </div>
-    </d.Content>
-  </ToolTip>
+  <input
+    aria-label="command input"
+    onkeyup={{action "handleKeyUp"}}
+    value={{this.value}}
+    autocomplete="off"
+    spellcheck="false"
+  />
+  <Hds::Button
+    class="hds-side-nav__icon-button"
+    {{on "click" (action "fullscreen")}}
+    title={{if this.isFullscreen "minimize" "maximize"}}
```

Comment:
Tooltip trigger didn't have valid label associated, so I removed it. With a title on the icon button sighted users can still see the text